portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1564,4 +1564,80 @@ definition {
 			title = "WC Title");
 	}
 
+	@description = "When the user clicks on My Sites or Asset Libraries tabs within the modal, the list of Sites or Asset Libraries to which the user has NO access is NOT displayed"
+    @priority = "5"
+    test SitePermissions {
+        property portal.acceptance = "true";
+
+        ApplicationsMenu.gotoPortlet(
+            category = "Sites",
+            panel = "Control Panel",
+            portlet = "Sites");
+
+        Site.addBlankCP(
+            membershipType = "Private",
+            siteName = "Blank Site Name");
+
+        ProductMenu.gotoPortlet(
+            category = "Site Builder",
+            portlet = "Pages");
+
+        SitePages.addPublicPage(
+            pageName = "Site Page Name",
+            sitePageType = "Widget Page");
+
+        Navigator.gotoSitePage(
+            pageName = "Site Page Name",
+            siteName = "Blank Site Name");
+
+        Portlet.addPG(portletName = "Message Boards");
+
+        Navigator.gotoSitePage(
+            pageName = "Site Page Name",
+            siteName = "Blank Site Name");
+
+        MessageboardsThread.addPGWithPermissions(
+            threadBody = "MB Thread Message Body",
+            threadSubject = "MB Thread Message Subject",
+            viewableBySelection = "Owner");
+
+        ApplicationsMenu.gotoPortlet(
+            category = "Users",
+            panel = "Control Panel",
+            portlet = "Users and Organizations");
+			
+        User.addCP(
+            userEmailAddress = "userea@liferay.com",
+            userFirstName = "userfn",
+            userLastName = "userln",
+            userScreenName = "usersn");
+
+        ApplicationsMenu.gotoPortlet(
+            category = "Users",
+            panel = "Control Panel",
+            portlet = "Users and Organizations");
+
+        User.editPasswordCP(
+            requirePasswordReset = "true",
+            userEmailAddress = "userea@liferay.com",
+            userScreenName = "usersn");
+
+        User.logoutPG();
+
+            User.loginUserPG(
+            password = "test",
+            userEmailAddress = "userea@liferay.com");
+			
+		Click(locator1 = "UserBar#USER_AVATAR_IN_NAVIGATION_USER_PERSONAL_BAR_PORTLET");	
+
+        Click(locator1 = "UserBar#USER_DASHBOARD");
+
+        AssertElementNotPresent(
+           key_nodeName = "Blank Site Name",
+           locator1 = "Treeview#NODE_ITEM");
+
```

Comment:
Why you do the check here? Isn't it supposed that you have to check it in the Site and Asset Library's modal within the Content Dashboard?